Carnap/src/Carnap/Languages/PurePropositional/Logic/Ripley.hs

```diff
@@ -89,7 +95,7 @@ parseRipleyLNJ rtc n _ = do
 
   
 parseRipleyLNJProof ::  RuntimeNaturalDeductionConfig PurePropLexicon (Form Bool) -> String -> [DeductionLine RipleyLNJ PurePropLexicon (Form Bool)]
-parseRipleyLNJProof rtc = toDeductionLemmon (parseRipleyLNJ rtc) (purePropFormulaParser ripleyOpts)
+parseRipleyLNJProof rtc = toDeductionLemmonImplicit (parseRipleyLNJ rtc) (purePropFormulaParser ripleyOpts)
```

Comment:
Simple `toDeductionLemmon` results in Lemmon style proofs where assumpions need to be explicitly discharged by being cited in the justification of the line, so that lines might look like:
```
[3] P AS
...
[8] P->Q [3](7) ->E
```

 I thought you might want a lemmon proof where you only need to update the dependencies correctly, and don't need to explicitly cite the assumptions that you're discharging, so it'd look like
```
[3] P AS
...
[8] P->Q (7) ->E
```